deploy/examples/filesystem.yaml

```diff
@@ -1,6 +1,8 @@
 #################################################################################################################
 # Create a filesystem with settings with replication enabled for a production environment.
-# A minimum of 3 OSDs on different nodes are required in this example.
+# A minimum of 3 OSDs on different nodes are required in this example. It allows only one
+# mds daemon per node, which may be too restrictive if you create more than one filesystem
```

Comment:
What about simplifying to a single line, something like this? I like simple headers. :)
```suggestion
# If one mds daemon per node is too restrictive, see the podAntiAffinity below. 
```